featuretools/entityset/deserialize.py

```diff
@@ -108,92 +52,66 @@ def description_to_entityset(description, **kwargs):
-    '''Read description data from disk.
-
-    Args:
-        description (dict) : Description of :class:`.Entity`.
-        path (str): Location on disk to read entity data.
-
-    Returns:
-        df (DataFrame) : Instance of dataframe.
-    '''
-    file = os.path.join(path, description['loading_info']['location'])
-    kwargs = description['loading_info'].get('params', {})
-    load_format = description['loading_info']['type']
-    entity_type = description['loading_info'].get('entity_type', 'pandas')
-    dtypes = description['loading_info'].get('properties', {}).get('dtypes')
-    read_kwargs = {}
-    if entity_type == 'dask':
-        lib = dd
-        read_kwargs['dtype'] = dtypes
-    elif entity_type == 'koalas':
-        import_error = 'Cannot load Koalas entityset - unable to import Koalas. ' \
-                       'Consider doing a pip install with featuretools[koalas] to install Koalas with pip'
-        lib = import_or_raise('databricks.koalas', import_error)
-        read_kwargs['multiline'] = True
-        kwargs['compression'] = str(kwargs['compression'])
-    else:
-        lib = pd
-    if load_format == 'csv':
-        dataframe = lib.read_csv(
-            file,
-            engine=kwargs['engine'],
-            compression=kwargs['compression'],
-            encoding=kwargs['encoding'],
-            **read_kwargs
-        )
-    elif load_format == 'parquet':
-        dataframe = lib.read_parquet(file, engine=kwargs['engine'])
-    elif load_format == 'pickle':
-        dataframe = pd.read_pickle(file, **kwargs)
-    else:
-        error = 'must be one of the following formats: {}'
-        raise ValueError(error.format(', '.join(FORMATS)))
-    if entity_type == 'koalas':
-        for col, dtype in dtypes.items():
-            if dtype == 'object':
-                dtypes[col] = 'str'
-            if dtype == 'datetime64[ns]':
-                dtypes[col] = np.datetime64
-    dataframe = dataframe.astype(dtypes)
-
-    if load_format in ['parquet', 'csv']:
-        latlongs = []
-        for var_description in description['variables']:
-            if var_description['type']['value'] == LatLong.type_string:
-                latlongs.append(var_description["id"])
-
-        def parse_latlong_tuple(x):
-            return tuple(float(y) for y in x[1:-1].split(","))
-
-        def parse_latlong_list(x):
-            return list(float(y) for y in x[1:-1].split(","))
-
-        for column in latlongs:
-            if entity_type == 'dask':
-                meta = (column, tuple([float, float]))
-                dataframe[column] = dataframe[column].apply(parse_latlong_tuple,
-                                                            meta=meta)
-            elif entity_type == 'koalas':
-                dataframe[column] = dataframe[column].apply(parse_latlong_list)
-
+    # TODO: Can we update Woodwork to return an empty initialized dataframe from a description
+    # instead of using this function?
```

Comment:
I feel like the empty dataframe might be a featuretools specific request (I'm still kind of wondering if we can find a way to just use a TableSchema here), but we could consider having the part of this function that's shared with woodwork's `_typing_information_to_woodwork_table` be a separate function that returns a TableSchema that can be used here to init with the empty dataframe. 

Not sure if initializing with a schema might pose problems, though

Reply:
It's probably not a big deal either way, but this code is more or less duplicated from Woodwork and thought it might be nice to not have to duplicate it here. Maybe not something we need to worry too much about in the short term though.